test/openAPI/features/wallet_consent.feature

```diff
@@ -0,0 +1,33 @@
+@method=POST @endpoint=/linked-authorization/consent
+Feature: The endpoint to send the accepted user claims and permitted scopes.
+
+  @smoke @unit @positive
+  Scenario: Successfully sends the accepted user claims and permitted scopes smoke type test
+    Given Wants to send the accepted user claims and permitted scopes
+    And The linkedTransactionId is generated before POST /linked-authorization/consent
+    When Send POST /linked-authorization/consent request with given requestTime and linkedTransactionId
+    Then Receive a response from the /linked-authorization/consent endpoint
+    And The /linked-authorization/consent endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/consent endpoint response should have status 200
```

Comment:
Why the below checks are missed ?
    And The /linked-authorization/consent endpoint response should have content-type: application/json header
    And The /linked-authorization/consent endpoint response should contain linkedTransactionId
    And The /linked-authorization/consent endpoint response should match json schema without errors

Reply:
Because this is just smoke test and below is positive test so no need to add this steps